lib/services/ios-project-service.ts

```diff
@@ -844,6 +848,41 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		return Promise.resolve();
 	}
 
+	public checkForChanges(changesInfo: IProjectChangesInfo, options: IProjectChangesOptions, projectData: IProjectData): void {
+		const nextCommandProvisionUUID = options.provision;
+
+		if (typeof nextCommandProvisionUUID === "string") {
+			const pbxprojPath = this.getPbxProjPath(projectData);
+
+			// Check if the native project's signing is set to the provided provision...
+			let signingChanged = false;
+			if (this.$fs.exists(pbxprojPath)) {
+				const xcode = Xcode.open(pbxprojPath);
+				const signing = xcode.getSigning(projectData.projectName);
+				if (signing && signing.style === "Manual") {
```

Comment:
It is `"Manual" | "Automatic"`, the TypeScript compiler does enough of type checking for these types. E.g. it will error if you compare it with "Manual1".